### DISCUSSION TITLE
Can I submit a PR to repo? Which branch should I submit to?

### DISCUSSION THREAD

**Discussion by DarkAngelZT** [bitbrain/beehave#330] [Q&A] [answered]
While I'm coding my own little experimental game, I've made a simple parallel composite node like the one in Unreal Engine. It's written in gdscript, and I think some one probably want this feature very bad. So I wonder if I can submit a PR. And which branch right now should I submit my code to?

**Comment by bitbrain** [comment, marked as answer]
`godot-4.x` is the branch for Godot 4.
